actions/check-product-changes/index.js

```diff
@@ -16,13 +16,13 @@ const { StateManager } = require('./lib/state');
 
 async function main(params) {
   const logger = Core.Logger('main', { level: params.LOG_LEVEL || 'info' });
-  const stateLib = await State.init();
-  const filesLib = await Files.init();
+  const stateLib = await State.init(params.libInit || {});
+  const filesLib = await Files.init(params.libInit || {});
   const stateMgr = new StateManager(stateLib, { logger });
 
   const running = await stateMgr.get('running');
   if (running?.value === 'true') {
-    return { state: 'skipped' };
+    //return { state: 'skipped' };
```

Comment:
This is needed in order to
1) skip execution if another activation is still in progress (no concurrency)
2) provide signalling about skipping the execution

Reply:
This line will be uncommented, this is only for local development